aten/src/ATen/core/op_registration/kernel_functor_test.cpp

```diff
@@ -429,13 +429,13 @@ TEST(OperatorRegistrationTest_FunctorBasedKernel, givenKernelWithIntListInput_wi
   auto op = c10::Dispatcher::singleton().findSchema("_test::int_list_input", "");
   ASSERT_TRUE(op.has_value());
 
-  auto outputs = callOp(*op, dummyTensor(TensorType1()), IntList::create({2, 4, 6}));
+  auto outputs = callOp(*op, dummyTensor(TensorType1()), IntList::create(c10::make_list<int64_t>({2, 4, 6})));
```

Comment:
for cases like this - can we just overload `create` to take initializer list?

Reply:
This is fixed in https://github.com/pytorch/pytorch/pull/21317